vent/io/devices/sensors.py

```diff
@@ -262,6 +260,41 @@ def _check_and_set_attr(self, **kwargs):
         self._fill_attr()
 
 
+class DLiteSensor(AnalogSensor):
+    """ D-Lite flow sensor setup.
+    This consists of the GE D-Lite sensor configured with
+    vacuum lines running to an analog differential pressure sensor.
+
+    """
+    def __init__(self, adc, **kwargs):
+
+        super().__init__(adc, **kwargs)
+
+    def _convert(self, raw) -> float:
+        """ Converts the raw differential voltage signal to
+        a measurement of flow in liters-per-minute (LPM).
+        
+        We calibrate the D-Lite flow readings using the
+        (pre-calibrated) Sensirion flow sensor (see SFM3200). 
+
+        Args:
+            raw (float): The raw sensor reading to convert.
+        """
+        raw = super()._convert(raw)
+        fit_param = 6.0345e-05
+        if(raw >= 0):

```

Comment:
dk what's going on in this method

Reply:
That is Julie's regression fit for the differential pressure based sensor (uses one of [these](https://www.gehealthcare.com/-/jssmedia/ae63d7860ee64d23833eae6ef48ef740.pdf?la=en-us)) 

The actual fit is a WIP so this whole class is more of a prototype.